apps/e2e/tests/backend/endpoints/api/v1/internal-metrics.test.ts

```diff
@@ -0,0 +1,433 @@
+import { describe } from "vitest";
+import { createMailbox, it } from "../../../../helpers";
+import { Auth, backendContext, niceBackendFetch, Project } from "../../../backend-helpers";
+
+describe("/internal/metrics route", () => {
```

Comment:
Can you test some more things? eg.:

- If user 1 signs in, then user 2 signs in, then user 1 signs in again, that user 1 is listed for in recently active users? (might wanna pass an email argument to the createMailbox() that's not a random UUID so it doesn't get stripped anymore)
- That this doesn't work without admin access type
- That it works on newly created projects
- That it returns exactly 10 recent signups, even when there have been more than ten signups (I proposed a comment on how to do this below)

Reply:
- [ ] If user 1 signs in, then user 2 signs in, then user 1 signs in again, that user 1 is listed for in recently active users? (might wanna pass an email argument to the createMailbox() that's not a random UUID so it doesn't get stripped anymore)
- [x] That this doesn't work without admin access type
- [x] That it works on newly created projects
- [x]That it returns exactly 10 recent signups, even when there have been more than ten signups (I proposed a comment on how to do this below)